cmd/ocsp-responder/main.go

```diff
@@ -27,6 +27,23 @@ import (
 	"github.com/letsencrypt/boulder/sa"
 )
 
+type statsShim struct {
+	responseTypes *prometheus.CounterVec
+}
+
+var responseTypeToString = map[ocsp.ResponseStatus]string{
+	ocsp.Success:           "Success",
+	ocsp.Malformed:         "Malformed",
+	ocsp.InternalError:     "InternalError",
+	ocsp.TryLater:          "TryLater",
+	ocsp.SignatureRequired: "SignatureRequired",
+	ocsp.Unauthorized:      "Unauthorized",
+}
+
+func (ss *statsShim) ResponseStatus(status ocsp.ResponseStatus) {
```

Comment:
Do you think its worth checking `_, ok := responseTypeToString[status]; !ok` and using a type label like `"unknown"`? I think it would end up producing a `"type": ""` in the unlikely event that `responseTypeToString` isn't exhaustive due to a change/bug with upstream.

Reply:
Done.